tests/wolfssl_crypto_test/Makefile

```diff
@@ -0,0 +1,47 @@
+APPLICATION = wolf_crypto_test
+
+# If no BOARD is found in the environment, use this default:
+#BOARD ?= native
+#BOARD ?= samd21-xpro #Currently missing implementation for strncasecmp
+#BOARD ?= pic32-wifire
+#BOARD ?= frdm-k64f
+BOARD ?= stm32f4discovery
+
+# BOARD_WHITELIST := To Be Determined
+# BOARD_BLACKLIST := To Be Determined
+# BOARD_INSUFFICIENT_MEMORY := To Be Determined
```

Comment:
same as above

Reply:
this is also fixed (see new blacklist in the unified test app)